etna/transforms/timestamp/holiday.py

```diff
@@ -7,14 +8,45 @@
 import numpy as np
 import pandas as pd
 
+from etna.datasets import TSDataset
 from etna.transforms.base import IrreversibleTransform
 
 
+def bigger_than_day(freq):
```

Comment:
This solution seems hacky. Have you tried smth more in line with pandas API?

Reply:
I can't find something useful in pandas API to solve this task.